### ISSUE TITLE
Close msg

### ISSUE THREAD

**Issue by dbrgn** [saltyrtc/saltyrtc-meta#82]
First, the spec states:

> However, this message SHALL ONLY be sent in case the client-to-client handshakes has been completed

(Btw, missing period there.)

But later the following line is there:

> The client SHALL also terminate the connection to the server with a close code of 1001 (Going Away).

In the case of a successful handover, there is no server connection.

Do we need rewording, or is it fine like this?

**Comment by lgrahl** [comment]
I don't really understand the issue here. Could you rephrase?

**Comment by dbrgn** [comment]
There is a high probability that when a close message is sent or received, that there _is_ no server connection. Yet the spec requires the user to close the server connection with a certain close code.

**Comment by lgrahl** [comment]
But why would you need the server connection to send a 'close' message?

**Comment by lgrahl** [comment]
We could change that to

> [...] The client SHALL also terminate the connection to the server with a close code of `1001` (_Going Away_) if a connection exists.

That might be confusing. What do you think?

**Comment by dbrgn** [comment]
What about this?

> [...] The client SHALL also terminate the connection to the server with a close code of 1001 (Going Away) if the connection is still open.

**Comment by lgrahl** [comment]
OK. Fine by me.
